sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3115,6 +3154,25 @@ def teardown_no_lock(self,
                 # successfully removed cluster as no exception was raised
                 returncode = 0
 
+        elif terminate and isinstance(cloud, clouds.SCP):
+            config['provider']['cache_stopped_nodes'] = not terminate
+            provider = SCPNodeProvider(config['provider'], handle.cluster_name)
+            try:
+                if not os.path.exists(provider.metadata.path):
+                    raise SCPError('SKYPILOT_ERROR_NO_NODES_LAUNCHED: '
+                                   'Metadata file does not exist.')
+
+                with open(provider.metadata.path, 'r') as f:
+                    metadata = json.load(f)
+                    node_id = next(iter(metadata.values())).get(
+                        'creation', {}).get('virtualServerId', None)
+                    provider.terminate_node(node_id)
+                returncode = 0
+            except SCPError as e:
+                returncode = 1
+                stdout = f'{e}'
+                stderr = f'{e}'
```

Comment:
nit: it would be enough to have the exception in one of the stdout/stderr.
```suggestion
                stdout = ''
                stderr = str(e)
```